README.md

```diff
@@ -194,7 +194,7 @@ scalability in a cluster.
 We designed [Modin's architecture](https://modin.readthedocs.io/en/latest/development/architecture.html)
 to be modular so we can plug in different components as they develop and improve:
 
-<img src="docs/img/modin_architecture.png" alt="Modin's architecture" width="75%"></img>
+<img src="https://github.com/modin-project/modin/raw/master/docs/img/modin_architecture.png" alt="Modin's architecture" width="75%"></img>
```

Comment:
diff looks wrong in the markdown preview for this PR, but architecture image looks fine on my branch: https://github.com/mvashishtha/modin/tree/5292-fix-make-readme-image-links-raw